include/tvm/ir/attrs.h

```diff
@@ -237,7 +237,7 @@ class DictAttrs : public Attrs {
    * \param dict The attributes.
    * \return The dict attributes.
    */
-  TVM_DLL explicit DictAttrs(Map<std::string, ObjectRef> dict);
+  TVM_DLL explicit DictAttrs(Map<String, ObjectRef> dict);
```

Comment:
NOTE: directly change Map<std::string, Object> -> Map<String, Object> will certainly cause problems in the system